  # Array of tones defined by frequency and duration
Tones[] score
